Environmental/new_crawlers/Weather forecast/anta_weather_forecast/consumer.py

```diff
@@ -0,0 +1,48 @@
+import os
+import json
+from kafka import KafkaConsumer
+from water_quality_cork.elastic import ElasticSearchClient
```

Comment:
i don't like this much...

either we specify some modules that are to be used by all producers/consumers or each producer/consumer pair has its own functionality